src/ui/screens/Council/CandidateScreen.tsx

```diff
@@ -98,7 +98,11 @@ export function CandidateScreen({route, navigation}: ScreenProps) {
                     left={() => <LeftIcon icon="twitter" />}
                     right={() => (
                       <ItemRight>
-                        <Caption>{twitter}</Caption>
+                        <Caption
+                          style={styles.twitter}
+                          onPress={() => Linking.openURL('https://twitter.com/' + twitter)}>
```

Comment:
Nitpick: Please use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) instead of concatenation.